add-on/panels/js/conversation.jsx

```diff
@@ -195,7 +195,7 @@ loop.conversation = (function(mozL10n) {
         textChatStore: textChatStore
       });
 
-      React.render(
+      ReactDOM.render(
```

Comment:
You need to change the stub in conversation_test.js to account for this change. There's probably similar stubs that may need updating in panels_test.js and webapp_test.js (and maybe other places)